packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -258,6 +258,14 @@ class CreateCommand extends CreateBase {
         'template: the language will always be C or C++.',
         exitCode: 2,
       );
+    } else if (!generateModule && argResults!.wasParsed('ios-language')) {
+      // Swift is not yet supported for add-to-app https://github.com/flutter/flutter/issues/23955.
```

Comment:
Super nit: this comment sounds like we don't support Swift in the host app, but we have docs showing you how to do that: https://docs.flutter.dev/add-to-app/ios/add-flutter-screen

Maybe: 

```suggestion
      // Swift is not yet supported for add-to-app modules https://github.com/flutter/flutter/issues/23955.
```

Reply:
I just removed that comment, I changed my mind to exclude the module template since that flag is ignored, so they _definitely_ should stop using it.